### PR TITLE
sssd aport

### PR DESCRIPTION
**Pull request by HRio**

### CODE REVIEW

**Comment by HRio** [comment]
@clandmeter & @rnalrd what do you think? good enough for testing/ ?

**Comment by rnalrd** [comment]
Besides these minor things, LGTM

**Comment by HRio** [comment]
@rnalrd thanks, fixing

**Comment by HRio** [comment]
@rnalrd even-tough there are a few todos and probably musl related unit test problems (not including all check depends due to this), do you think its good enough for testing after my latest changes? thanks.

File: testing/sssd/APKBUILD [hunk 1]
```diff
@@ -47,25 +53,27 @@ makedepends="autoconf \
 	openldap-dev \
 	krb5-dev \
         keyutils-dev \
-	check \
 	"
```
Comment (rnalrd) [inline testing/sssd/APKBUILD]: APKBUILDs variables don't need a backslash when there's a line break

File: testing/sssd/APKBUILD [hunk 2]
```diff
@@ -47,25 +53,27 @@ makedepends="autoconf \
...
-subpackages=""
+subpackages="$pkgname-dev $pkgname-openrc"
 source="https://releases.pagure.org/SSSD/$pkgname/$pkgname-$pkgver.tar.gz \
 	0001-makefile.patch \
 	0002-src.patch \
 	0003-alpine.patch \
-	nss.h cifsidmap.h "
+	nss.h \
+	cifsidmap.h \
+	"
```
Comment (rnalrd) [inline testing/sssd/APKBUILD]: Same here

**Comment by rnalrd** [comment]
Looks ok for besides cosmetic fixes on APKBUILD's variables line breaks. Please see code comments.

File: testing/sssd/APKBUILD (hunk 1 above)
Comment (HRio) [reply testing/sssd/APKBUILD]: thanks, fixing

File: testing/sssd/APKBUILD (hunk 2 above)
Comment (HRio) [reply testing/sssd/APKBUILD]: thanks, fixing

**Pull request closed without merging**